app/src/main/java/com/infomaniak/drive/ui/publicShare/PublicShareListFragment.kt

```diff
@@ -154,9 +165,11 @@ class PublicShareListFragment : FileListFragment() {
 
     private fun observeRootFile() {
         publicShareViewModel.rootSharedFile.observe(viewLifecycleOwner) { file ->
+            publicShareViewModel.fileClicked = file
```

Comment:
Is-it really useful ?

Reply:
Yes otherwise the multiselection will crash with a NPE